### ISSUE TITLE
Counting with decimal number

### ISSUE THREAD

**Issue by vlkous** [josdejong/mathjs#815]
I try counting this 0.1 + 100 - 100 and result is 0.099999999999994

Have you any idea how to fix that?

**Comment by josdejong** [comment]
Yes, see docs: http://mathjs.org/docs/datatypes/numbers.html#roundoff-errors
